### ISSUE TITLE
Crash on exporting geo-referenced pdf

### ISSUE THREAD

**Issue by NHM55** [qgis/QGIS#55208]
## User Feedback

Crashed when exporting geo-referencd pdf

## Report Details

**Python Stack Trace**
```
Windows fatal exception: code 0x8001010d

Current thread 0x00005b84 (most recent call first):
<no Python frame>
Windows fatal exception: code 0x800706ba

Thread 0x00005b84 (most recent call first):
<no Python frame>
Windows fatal exception: code 0x800706ba

Thread 0x00005b84 (most recent call first):
<no Python frame>
Windows fatal exception: code 0x800706ba

Thread 0x00005b84 (most recent call first):
<no Python frame>
Windows fatal exception: access violation

Thread 0x00005b84 (most recent call first):
<no Python frame>
```

**Stack Trace**

```

QgsMapRendererStagedRenderJob::renderCurrentPart :
QgsMapRendererTask::run :
QgsTask::start :
QThreadPoolPrivate::reset :
QThread::start :
BaseThreadInitThunk :
RtlUserThreadStart :
```

**QGIS Info**
QGIS Version: 3.34.0-Prizren
QGIS code revision: ffbdd67881
Compiled against Qt: 5.15.3
Running against Qt: 5.15.3
Compiled against GDAL: 3.7.2
Running against GDAL: 3.7.2

**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.19045

### Steps to reproduce the issue

Program crashes after setup, file name save, etc.

### Versions

<html><body>
&lt;style type="text/css"&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;

QGIS version | 3.34.0-Prizren | QGIS code revision | ffbdd67
-- | -- | -- | --
Qt version | 5.15.3 |   |  
Python version | 3.9.5 |   |  
GDAL/OGR version | 3.7.2 |   |  
PROJ version | 9.3.0 |   |  
EPSG Registry database version | v10.094 (2023-08-08) |   |  
GEOS version | 3.12.0-CAPI-1.18.0 |   |  
SQLite version | 3.41.1 |   |  
PDAL version | 2.5.5 |   |  
PostgreSQL client version | 15.2 |   |  
SpatiaLite version | 5.1.0 |   |  
QWT version | 6.1.6 |   |  
QScintilla2 version | 2.13.4 |   |  
OS version | Windows 10 Version 2009 |   |  
  |   |   |  
Active Python plugins |   |   |  
AzimuthDistanceCalculator | 2.0.2 |   |  
ee_plugin | 0.0.6 |   |  
FreehandRasterGeoreferencer | 0.8.3 |   |  
GeometryShapes | 0.7 |   |  
getthemfiltered | 0.2.0 |   |  
GroupStats | 2.2.7 |   |  
ImportPhotos | 3.0.5 |   |  
indiana_ortho | 0.1.7 |   |  
LAStools | 1.5 |   |  
layerGroupFilter | 0.2 |   |  
LocatePoints | 0.3.4 |   |  
mmqgis | 2021.9.10 |   |  
numericalDigitize | 0.4.7 |   |  
pointsamplingtool | 0.5.4 |   |  
profiletool | 4.2.6 |   |  
qgis_resource_sharing | 1.0.0 |   |  
qgsAzimuth | 0.9.16 |   |  
qtiles | 1.7.2 |   |  
quick_map_services | 0.19.33 |   |  
raster_tracer | 0.3.3 |   |  
toggle_labels_widget | 1.0.2 |   |  
db_manager | 0.1.20 |   |  
grassprovider | 2.12.99 |   |  
MetaSearch | 0.3.6 |   |  
processing | 2.12.99 |   |  

QGIS version
3.34.0-Prizren
QGIS code revision
[<a class="commit-link" data-hovercard-type="commit" data-hovercard-url="https://github.com/qgis/QGIS/commit/ffbdd678812eec460ff02ff73bd36ee7955ac8a5/hovercard" href="https://github.com/qgis/QGIS/commit/ffbdd678812eec460ff02ff73bd36ee7955ac8a5"><tt>ffbdd67</tt></a>](https://github.com/qgis/QGIS/commit/ffbdd67881)
Qt version
5.15.3
Python version
3.9.5
GDAL/OGR version
3.7.2
PROJ version
9.3.0
EPSG Registry database version
v10.094 (2023-08-08)
GEOS version
3.12.0-CAPI-1.18.0
SQLite version
3.41.1
PDAL version
2.5.5
PostgreSQL client version
15.2
SpatiaLite version
5.1.0
QWT version
6.1.6
QScintilla2 version
2.13.4
OS version
Windows 10 Version 2009
<p dir="auto">Active Python plugins<br>
AzimuthDistanceCalculator<br>
2.0.2<br>
ee_plugin<br>
0.0.6<br>
FreehandRasterGeoreferencer<br>
0.8.3<br>
GeometryShapes<br>
0.7<br>
getthemfiltered<br>
0.2.0<br>
GroupStats<br>
2.2.7<br>
ImportPhotos<br>
3.0.5<br>
indiana_ortho<br>
0.1.7<br>
LAStools<br>
1.5<br>
layerGroupFilter<br>
0.2<br>
LocatePoints<br>
0.3.4<br>
mmqgis<br>
2021.9.10<br>
numericalDigitize<br>
0.4.7<br>
pointsamplingtool<br>
0.5.4<br>
profiletool<br>
4.2.6<br>
qgis_resource_sharing<br>
1.0.0<br>
qgsAzimuth<br>
0.9.16<br>
qtiles<br>
1.7.2<br>
quick_map_services<br>
0.19.33<br>
raster_tracer<br>
0.3.3<br>
toggle_labels_widget<br>
1.0.2<br>
db_manager<br>
0.1.20<br>
grassprovider<br>
2.12.99<br>
MetaSearch<br>
0.3.6<br>
processing<br>
2.12.99</p>
</body>
</html><style type="text/css"> p, li { white-space: pre-wrap; } </style>
QGIS version 	3.34.0-Prizren 	QGIS code revision 	https://github.com/qgis/QGIS/commit/ffbdd678812eec460ff02ff73bd36ee7955ac8a5
Qt version 	5.15.3 		
Python version 	3.9.5 		
GDAL/OGR version 	3.7.2 		
PROJ version 	9.3.0 		
EPSG Registry database version 	v10.094 (2023-08-08) 		
GEOS version 	3.12.0-CAPI-1.18.0 		
SQLite version 	3.41.1 		
PDAL version 	2.5.5 		
PostgreSQL client version 	15.2 		
SpatiaLite version 	5.1.0 		
QWT version 	6.1.6 		
QScintilla2 version 	2.13.4 		
OS version 	Windows 10 Version 2009 		
  	  	  	 
Active Python plugins 			
AzimuthDistanceCalculator 	2.0.2 		
ee_plugin 	0.0.6 		
FreehandRasterGeoreferencer 	0.8.3 		
GeometryShapes 	0.7 		
getthemfiltered 	0.2.0 		
GroupStats 	2.2.7 		
ImportPhotos 	3.0.5 		
indiana_ortho 	0.1.7 		
LAStools 	1.5 		
layerGroupFilter 	0.2 		
LocatePoints 	0.3.4 		
mmqgis 	2021.9.10 		
numericalDigitize 	0.4.7 		
pointsamplingtool 	0.5.4 		
profiletool 	4.2.6 		
qgis_resource_sharing 	1.0.0 		
qgsAzimuth 	0.9.16 		
qtiles 	1.7.2 		
quick_map_services 	0.19.33 		
raster_tracer 	0.3.3 		
toggle_labels_widget 	1.0.2 		
db_manager 	0.1.20 		
grassprovider 	2.12.99 		
MetaSearch 	0.3.6 		
processing 	2.12.99 		
QGIS version 3.34.0-Prizren QGIS code revision [https://github.com/qgis/QGIS/commit/ffbdd678812eec460ff02ff73bd36ee7955ac8a5](https://github.com/qgis/QGIS/commit/ffbdd67881) Qt version 5.15.3 Python version 3.9.5 GDAL/OGR version 3.7.2 PROJ version 9.3.0 EPSG Registry database version v10.094 (2023-08-08) GEOS version 3.12.0-CAPI-1.18.0 SQLite version 3.41.1 PDAL version 2.5.5 PostgreSQL client version 15.2 SpatiaLite version 5.1.0 QWT version 6.1.6 QScintilla2 version 2.13.4 OS version Windows 10 Version 2009

Active Python plugins
AzimuthDistanceCalculator
2.0.2
ee_plugin
0.0.6
FreehandRasterGeoreferencer
0.8.3
GeometryShapes
0.7
getthemfiltered
0.2.0
GroupStats
2.2.7
ImportPhotos
3.0.5
indiana_ortho
0.1.7
LAStools
1.5
layerGroupFilter
0.2
LocatePoints
0.3.4
mmqgis
2021.9.10
numericalDigitize
0.4.7
pointsamplingtool
0.5.4
profiletool
4.2.6
qgis_resource_sharing
1.0.0
qgsAzimuth
0.9.16
qtiles
1.7.2
quick_map_services
0.19.33
raster_tracer
0.3.3
toggle_labels_widget
1.0.2
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.6
processing
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

### Additional context

I have also submitted a bug report #55204. I think the 'incorrect decoration placement' and the 'crash on pdf export' problems are related.

**Comment by agiudiceandrea** [comment]
@NHM55, thanks for reporting. Could you please try if the issue does occur also actually using a new QGIS user profile without any external plugin installed? If yes, please provide a sample project and layer with exact step and parameter in order to replicate the issue.

**Comment by NHM55** [comment]
Andrea,

Thanks for your assistance. Here is what I have done so far:

- I set up a new profile as requested. Here are the loaded plugins:

[image.png]

- I then loaded a project file (the same file I was using when the 'Export pdf' crash occurred this morning). Upon loading, I can see that the the Decorations are still in the wrong location. 😕
- I tried another 'Export to pdf' and this time it worked! 🙂

I still don't understand why the Decorations (N arrow & Scale bar) are misplaced. Regarding the export feature, which worked as expected, I notice the "Freehand raster georeferencer" plugin is no longer present. (I tried reloading, etc. but can't seem to make it functional. ??? ) This particular plugin has been causing a small problem with 'Save Project' function for some time. Could this particular plugin have caused the earlier crash?

Bottom line:

- The Decorations function is still not working as expected. Placement on canvas is incorrect.
- 'Export to pdf' function appears to be working in absence of the 'Freehand raster georeferencer' plugin.

Chris

**Comment by agiudiceandrea** [comment]
From your last comment it looks like the issue is due to an external plugin. Please report the issue to the plugin authors using its issue tracker.
